### DISCUSSION TITLE
Search for mods with a x as their dependency

### DISCUSSION THREAD

**Discussion by KirboSoftware** [modrinth/.github#90] [Feature requests]
**Is your feature request related to a problem? Please describe.**
This would be a more QoL thing for on modrinth. Imagine looking through the page and you already have a mod installed, but that mod has addons. You want to look for those addon's but you can't find it as there is no easy way to look them. 

**Describe the solution you'd like**
Whenever you're looking for the base mod, that it could list the mods that depend on that mod on addition with the name and desciption scanning. 

**Describe alternatives you've considered**
Maybe if you click the dependency of said mod that you could have a overview of what mods have that dependency. 

**Additional context**
You're looking for mods as example that make use of trinkets. You seach [trinkets](https://modrinth.com/mods?q=trinkets&g=categories%3A%27fabric%27) or [trinket](https://modrinth.com/mods?q=trinket&g=categories%3A%27fabric%27) and get a overview of mods with the name or have trinkets/trinket in their description. But you would be missing out on [atrifacts](https://modrinth.com/mod/artifacts/version/7.1.1%2Bfabric) as example while they do have trinkets as their dependency but not in their name or description.

**Comment by BlueZeeKing** [comment]
Another tab on the navigation bar that contains the description, versions, and the changelog could be added with a list of dependents

**Comment by quat1024** [comment]
Would be very cool to search for modpacks that contain a given mod, too. CurseForge has this, but, naturally, blends all the mods and modpacks together into a giant ball lmao

**Reply by KirboSoftware** [reply]
I mean, it all ties to be able to see dependencies/dependents on modrint

**Reply by gdavid04** [reply]
A search filter that allows specifying multiple dependencies would be better for this imo. Seeing what modpacks contain a set of mods is much more useful than having to search for packs containing each individual mod separately.
